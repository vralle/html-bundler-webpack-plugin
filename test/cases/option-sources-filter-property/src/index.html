<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <meta property="og:title" content="Apple" />
    <meta property="og:image" content="@images/apple.png" />
    <meta property="og:image:type" content="image/png" />
    <meta property="og:video" content="@videos/video.mp4" />
    <meta property="og:video:type" content="video/mp4" />
    <!-- test: the `content` w/o `property` attribute must not be resolved -->
    <meta name="source" content="@images/apple.png">
  </head>
  <body>
    <h1>Home</h1>
    <img src="@images/apple.png">
  </body>
</html>