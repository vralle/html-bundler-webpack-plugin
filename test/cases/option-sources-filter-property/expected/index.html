<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <meta property="og:title" content="Apple" />
    <meta property="og:image" content="assets/img/apple.02a7c382.png" />
    <meta property="og:image:type" content="image/png" />
    <meta property="og:video" content="assets/video/video.697ef306.mp4" />
    <meta property="og:video:type" content="video/mp4" />
    <!-- test: the `content` w/o `property` attribute must not be resolved -->
    <meta name="source" content="@images/apple.png">
  </head>
  <body>
    <h1>Home</h1>
    <img src="assets/img/apple.02a7c382.png">
  </body>
</html>